test/test_preprocessing_encoders.py

```diff
@@ -254,3 +258,66 @@ def test_robust_ordinal_encoding_inverse_transform_floatkeys():
     assert reverse.dtype == object
     assert np.array_equal(data[1:], reverse[:-1])
     assert all([x is None for x in reverse[-1]])
+
+
+titanic = pd.read_csv("test/data/csv/titanic.csv")
```

Comment:
Let's use `sagemaker_sklearn_extension.externals.read_csv_data` instead of pandas